apps/storybook.namekit.io/stories/Namekit/DomainCardPrice.stories.tsx

```diff
@@ -0,0 +1,90 @@
+import {
+  Currency,
+  CurrencySymbology,
+  getCurrencySymbology,
+  PriceCurrencyFormat,
+} from "@namehash/ens-utils";
+import { CurrencySymbol, DomainCardPrice } from "@namehash/namekit-react";
+import type { Meta, StoryObj } from "@storybook/react";
+import React from "react";
+
+const meta: Meta<typeof DomainCardPrice> = {
+  component: DomainCardPrice,
+  title: "Namekit/DomainCardPrice",
+  argTypes: {
+    altPrice: {
+      control: "object",
+    },
+    altPriceSymbology: {
+      options: Object.keys(CurrencySymbology),
+      control: { type: "select" },
+    },
+  },
+};
+
+declare global {
+  interface BigInt {
+    toJSON(): string;
+  }
+}
+
+BigInt.prototype.toJSON = function () {
+  return `${this.toString()}n`;
+};
+
+export default meta;
+
+type Story = StoryObj<typeof DomainCardPrice>;
+
+export const EthPriceWithCurrencySymbolAndAnUsdAltPriceWithSymbolSymbology: Story =
+  {
+    args: {
+      price: { currency: Currency.Eth, value: 1000000000000000000n },
```

Comment:
Unless there's a special reason not to for a particular story, let's please use the `numberToPrice` function for all of the price values in all storybook stories. Then you can just just do `numberToPrice(1)` to get 1 ETH.

We should always use these utility functions to build our special types and never explicitly build them the way you are here. For example, we might change the utility functions to add special invariants in the future, but if you build them explicitly as you are here then it would create bugs and we might not notice for some time.